src/wallet/sqlite.h

```diff
@@ -166,6 +166,14 @@ class SQLiteDatabase : public WalletDatabase
     void IncrementUpdateCounter() override { ++nUpdateCounter; }
 
     std::string Filename() override { return m_file_path; }
+    /** Return paths to all database created files */
```

Comment:
Should probably be skipped for `BerkeleyDatabase`, but should have a test for `Files()`, e.g.:

Maybe something like https://github.com/davidgumberg/bitcoin/commit/1428104fe9c805554e17a1a177214033b8a02a93 if it makes sense to use an external library in testing like this.